tutorials/best_practices/godot_interfaces.rst

```diff
@@ -55,7 +55,10 @@ access.
     tool # Must place at top of file.
 
     # Must configure from the editor, defaults to null.
-    export(Script) var const_script setget set_const_script
+    @export var const_script: Script = null:
+        set(value):
+            set_const_script(value)
```

Comment:
Shouldn't it be this?
```gdscript
@export var const_script: Script = null:
    set = set_const_script
```

Reply:
One could argue that having the two line function set_const_script() isn't needed anyway I think.  But my thinking was to show how a value is passed via the property setter as omitting it is less clear to me what is happening.

Reply:
Well, it's new syntax, it'll take some time for it to become natural. But doing it the way you did comes with an unnecessary overhead, as you create an extra function just to pass the value. We have a way to directly assign a setter function in this syntax, we should use it.

Reply:
If the point is to reduce extra methods, wouldn't it be best to avoid the whole method set_const_script() and just put the conditional `if` inside the setter method altogether?

Reply:
That would be equal amount of methods to what Adam suggests.

Reply:
I've made the suggested changes